src/main/resources/static/styles/index.css

```diff
@@ -3,6 +3,7 @@ body {
     font-family: 'Montserrat', sans-serif;
     background-color: rgb(185 185 185);
     margin: 0;
+    word-break: break-all;

```

Comment:
I found "overflow-wrap: anywhere" which performs better - it does not cut single letters from words when there are spaces.